### PR TITLE
docs: esl panel and esl panel group components

### PR DESCRIPTION
**Pull request by julia-murashko**

### CODE REVIEW

File: src/modules/esl-panel/README.md [hunk 1]
```diff
@@ -6,16 +6,33 @@ Version: *1.0.0-beta*.
 Authors: *Julia Murashko*.
 
 ESLPanel - is a custom element, that is used as a wrapper for content that 
-can be shown/hidden or collapsed (animated height change).
+can be shown/hidden or collapsed (height animation).
```
Comment (dshovchko) [inline src/modules/esl-panel/README.md:9]: It seems to me that this option is slightly annoying. Try this:

ESLPanel - is a custom element, which is used as a wrapper for content that can be shown/hidden, or collapsed (height animation).

File: src/modules/esl-panel/README.md [hunk 2]
```diff
@@ -6,16 +6,33 @@ Version: *1.0.0-beta*.
...
+ - `post-animate-class` - classes to be added during animation after next render
+ - `fallback-duration` - time after which the animation will be cleared ('auto' by default)
+ - `initial-params` - initial params for current ESLPanel instance
+ 
+**Note:** List of other supported attributes you can find here [ESLToggleable](./../esl-toggleable/README.md).
+ 
+**ESLPanelGroup component**
+ 
+- `mode` - mode of the component. Takes values from list of supported modes ('accordion' by default).
+- `mode-cls-target` - target element [ESLTraversingQuery](./../esl-traversing-query/README.md) select to add mode classes
```
Comment (dshovchko) [inline src/modules/esl-panel/README.md:35]: Maybe `selector` instead `select`?

File: src/modules/esl-panel/core/esl-panel-group.ts [hunk 3]
```diff
@@ -7,20 +7,33 @@ import {ESLMediaRuleList} from '../../esl-media-query/core';
...
+ */
 @ExportNs('PanelGroup')
 export class ESLPanelGroup extends ESLBaseElement {
   public static is = 'esl-panel-group';
+  /** List of supported modes */
   public static supportedModes = ['tabs', 'accordion'];
 
+  /** Mode of the component (takes values from list of supported modes) */
   @attr({defaultValue: 'accordion'}) public mode: string;
+  /** Target element {@link TraversingQuery} select to add mode classes */
```
Comment (dshovchko) [inline src/modules/esl-panel/core/esl-panel-group.ts:23]: The same, selector instead select.

File: src/modules/esl-panel/core/esl-panel.ts [hunk 4]
```diff
@@ -10,25 +10,37 @@ export interface PanelActionParams extends ToggleableActionParams {
   noCollapse?: boolean;
 }
 
+/**
+ * ESLPanel - is a custom element, that is used as a wrapper for content that
+ * can be shown/hidden or collapsed (animated height change).
```
Comment (dshovchko) [inline src/modules/esl-panel/core/esl-panel.ts:15]: ESLPanel - is a custom element, which is used as a wrapper for content that can be shown/hidden, or collapsed (height animation).

File: src/modules/esl-panel/core/esl-panel.ts [hunk 5]
```diff
@@ -86,32 +102,36 @@ export class ESLPanel extends ESLToggleable {
...
+  /** Clean up the bits of animation */
   @bind
   protected _onTransitionEnd(e?: TransitionEvent) {
     if (!e || e.propertyName === 'max-height') {
       this.afterAnimate();
     }
   }
 
-  /** The panels use panel group config for actions */
+  /** Merge params that are used by panel group for actions */
```
Comment (dshovchko) [inline src/modules/esl-panel/core/esl-panel.ts:134]: Merge params that are used by the panel group for actions

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-panel/README.md (hunk 1 above)
Comment (ala-n) [reply src/modules/esl-panel/README.md]: updated

File: src/modules/esl-panel/core/esl-panel.ts (hunk 4 above)
Comment (ala-n) [reply src/modules/esl-panel/core/esl-panel.ts]: updated

File: src/modules/esl-panel/core/esl-panel.ts (hunk 5 above)
Comment (ala-n) [reply src/modules/esl-panel/core/esl-panel.ts:139]: updated

**Review by ala-n — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Pull request merged**